ote/src/cljs/ote/views/route/route_list.cljs

```diff
@@ -45,91 +51,152 @@
-            [ui/table-row-column {:style {:width "10%"}} (when available-to (time/format-date available-to))]
-            [ui/table-row-column {:class "hidden-xs hidden-sm " :style {:width "15%"}} (time/format-timestamp-for-ui (or modified created))]
-            [ui/table-row-column {:class "hidden-xs hidden-sm " :style {:width "13%"}}
-             [ui/icon-button {:href     "#"
-                              :on-click #(do
-                                           (.preventDefault %)
-                                           (e! (fp/->ChangePage :edit-route {:id id})))}
-              [ic/content-create]]
-             [delete-route-action e! row]]])
-         routes))]]])
+       (for [{::t-service/keys [id name] :as s} services]
+         ^{:key (str "link-service-id-" name "-" id)}
+         [:div
+          [ui/checkbox {:label name
+                        :id (str "checkbox-" name "-" id)
+                        :checked (service-linked-to-route s services-with-route)
+                        :on-click #(e! (route-list/->ToggleLinkInterfaceToService
+                                         id
+                                         (service-linked-to-route s services-with-route)))}]]))]
+    [:div
+     [:h4 (tr [:route-list-page :header-link-interface-to-service])]
+     [:p (tr [:route-list-page :desc-link-interface-to-service])]]))
 
-(defn routes [e! {operators :transport-operators-with-services :as app}]
-  (e! (route-list/->LoadRoutes))
-  (fn [e! {routes :routes-vector operator :transport-operator :as app}]
+(defn list-routes [e! public-routes draft-routes]
+  [:div {:style {:padding-top "20px"}}
+   [:h4 (tr [:route-list-page :header-route-drafts])]
+   [:p (tr [:route-list-page :desc-route-drafts])]
+   [:div.drafts
+    [route-table e! draft-routes (if (not (empty? draft-routes)) nil (tr [:route-list-page :no-draft-routes]))]]
+   [:h4 (tr [:route-list-page :header-public-routes])]
+   [:p (tr [:route-list-page :desc-public-routes])]
+   [:div.public
+    [route-table e! public-routes (if (not (empty? public-routes)) nil (tr [:route-list-page :no-public-routes]))]]])
+
+(defn routes [e! {operator :transport-operator
+                  operators :transport-operators-with-services
+                  :as app}]
+  (let [routes (get-in app [:routes :routes-vector])
+        services-with-route (get-in app [:routes :route-used-in-services])
+        public-routes (filter (fn [r]
+                                (true? (::transit/published? r)))
+                              routes)
+        draft-routes (filter (fn [r]
+                               (false? (::transit/published? r)))
+                             routes)
+        operator-services (to/current-operator-services app)
+        schedule-services (filter
+                            (fn [{::t-service/keys [sub-type] :as s}]
+                              (or (= :schedule sub-type) false))
```

Comment:
Implementation is ok, just a remark on alternative approach which does the same.
```suggestion
                              (when (= :schedule sub-type)))
```